test/data_map_test.exs

```diff
@@ -0,0 +1,162 @@
+defmodule DataMapTest do
+  use PowerAssert
+
+  test "Insert empty row must fail" do
```

Comment:
I'm not a pro, but shouldn't it be something like "Inserting empty rows..." or "Inserting an empty row..."?